### PR TITLE
add sp standard

### PR DESCRIPTION
**Pull request by maxnodereal**

### CODE REVIEW

File: docs/guide/storage-provider/introduction/standard.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+## Auth Methods
+Permission management and authentication mechanisms are essential for a decentralized storage network, ensuring that only authorized users can access specific storage objects. The Greenfield Network defines standard permission divisions and authorization processes, requiring SP nodes to implement detailed permission control and user authentication accordingly. SP should achieve following three authentication at least:
+* AuthTypeV1
+It requires users to use a private key to sign for authentication.
+* PersonalSign
+It is used to verify wallet personal signature from a certain dApp website.
+* OffChainAuth
+It is used for some dApps and users to store the “off chain auth” account key in SPs. Users can communicate with the SP without explicit signature for most interactions.
+
+For more information please refer to [here](https://greenfield.bnbchain.org/docs/api-sdk/storgae-provider-rest/referenece/gnfd_headers.html#authorization-header).
```
Comment (KeefeL) [inline docs/guide/storage-provider/introduction/standard.md:61]: We should use the new document website: https://docs.bnbchain.org/greenfield-docs/docs, this old one will be deprecated, and I think the relative path here will be much better.

File: docs/guide/storage-provider/introduction/standard.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+## P2P Protocol
+SP nodes form a P2P network where they probe node liveliness, coordinate file uploads, etc. Currently, there are two main definitions and scenarios for P2P messages.
+* [Ping service](https://greenfield.bnbchain.org/docs/guide/storage-provider/services/p2p.html#ping-service). SP nodes ping each other to detect whether the other party is online and the delay status
```
Comment (KeefeL) [inline docs/guide/storage-provider/introduction/standard.md:65]: The same here, please check and change these urls.

Comment (maxnodereal) [reply docs/guide/storage-provider/introduction/standard.md]: resolved

File: docs/guide/storage-provider/introduction/standard.md (hunk 1 above)
Comment (maxnodereal) [reply docs/guide/storage-provider/introduction/standard.md]: resolved

**Review by guagualvcha — APPROVED** [review]

**Review by KeefeL — APPROVED** [review]

**Pull request merged**